cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -471,6 +471,41 @@ action.cyences_notable_event_action.products = Sophos Endpoint Protection
 action.cyences_notable_event_action.teams = SOC
 
 
+[Sophos Endpoint Protection - Firewall Lost Connection to Sophos Central]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 2,17,32,47 * * * *
+description = This alert will trigger when a Firewall lost connection to Sophos Central. \
+\
+Data Collection - Sophos Central Add-on for Splunk (https://splunkbase.splunk.com/app/6186/)
+dispatch.earliest_time = -17m@m
```

Comment:
It is that important to be alerting every 15 minutes or can we do hourly??

Reply:
I think we can do it hourly to reduce the alerting as sometimes it has 1-2 events hourly for same host and location.